plugins/source/aws/policies/queries/cloudwatch/alarm_organization_changes.sql

```diff
@@ -0,0 +1,31 @@
+insert into aws_policy_results
+select
+  :'execution_time' as execution_time,
+  :'framework' as framework,
+  :'check_id' as check_id,
+  'Ensure a log metric filter and alarm exists for AWS Organizations changes (Automated)' as title,
+  account_id,
+  cloud_watch_logs_log_group_arn as resource_id,
+  case when
+      pattern = '{ ($.eventName = AcceptHandshake) '
+      || '|| ($.eventName = AttachPolicy) '
+      || '|| ($.eventName = CreateAccount) '
+      || '|| ($.eventName = CreateOrganizationalUnit) '
+      || '|| ($.eventName = CreatePolicy) '
+      || '|| ($.eventName = DeclineHandshake) '
+      || '|| ($.eventName = DeleteOrganization) '
+      || '|| ($.eventName = DeleteOrganizationalUnit) '
+      || '|| ($.eventName = DeletePolicy) '
+      || '|| ($.eventName = DetachPolicy) '
+      || '|| ($.eventName = DisablePolicyType) '
+      || '|| ($.eventName = EnablePolicyType) '
+      || '|| ($.eventName = InviteAccountToOrganization) '
+      || '|| ($.eventName = LeaveOrganization) '
+      || '|| ($.eventName = MoveAccount) '
+      || '|| ($.eventName = RemoveAccountFromOrganization) '
+      || '|| ($.eventName = UpdatePolicy) '
+      || '|| ($.eventName = UpdateOrganizationalUnit) }'
+      then 'pass'
```

Comment:
This query seems very fragile, If I am reading it correctly if a user were to put the events in a different order then it would fail